torch/_inductor/utils.py

```diff
@@ -90,6 +90,7 @@ def get_gpu_type():
 VarRanges = Dict[sympy.Expr, sympy.Expr]
 InputType = Optional[Union[torch.Tensor, int, torch.SymInt]]
 
+GPU_KERNEL_BIN_EXTS = {"cuda": "cubin", "xpu": "spv"}
```

Comment:
```suggestion
GPU_KERNEL_BIN_EXTS = {"cuda": ".cubin", "xpu": ".spv"}
```
You are missing a period

Reply:
Refined, thanks.